packages/eslint-plugin/docs/rules/return-await.mdx

```diff
@@ -9,15 +9,46 @@ import TabItem from '@theme/TabItem';
 >
 > See **https://typescript-eslint.io/rules/return-await** for documentation.
 
-This rule builds on top of the [`eslint/no-return-await`](https://eslint.org/docs/rules/no-return-await) rule.
-It expands upon the base rule to add support for optionally requiring `return await` in certain cases.
+In `async` functions, it is valid to return a promise-wrapped value or a value directly, both of which ultimately produce a promise with the same fulfillment value.
 
-The extended rule is named `return-await` instead of `no-return-await` because the extended rule can enforce the positive or the negative. Additionally, while the core rule is now deprecated, the extended rule is still useful in many contexts:
+```ts
+async function foo(): Promise<number> {
+  // value
+  return 42;
+}
+foo().then(x => console.log(x)); // logs 42
+
+async function bar(): Promise<number> {
+  // promise-wrapped value
+  return Promise.resolve(42);
+}
+bar().then(x => console.log(x)); // logs 42
+
+async function baz(): Promise<number> {
+  // value, since `await Promise.resolve(42)` is 42
+  return await Promise.resolve(42);
+}
+baz().then(x => console.log(x)); // logs 42
+```
```

Comment:
Here's how the new page looks on my monitor:

![Screenshot of the new docs page with the Netlify Drawer. The last fully visible sentence is the 'Returning a value rather than ...' one just after the code block.](https://github.com/user-attachments/assets/349c87a6-459f-498b-bde3-208b435fb97b)

Not much explanation above the fold, but a lot of code block. I think that's a tl;dr for many users.

Suggestion: what do you think about moving the plain text descriptions above the code block?

Reply:
I'm not sure exactly what you mean by "moving the plain text descriptions above the code block"... 

However, I removed the code block for now, and I think it reads ok. What do you think?

Reply:
That's what I meant yeah - moving more of the text explanations above the code block. 👍